TurkishFilter/sections/adservers.txt

```diff
@@ -25,6 +25,7 @@
 ||bumbook.xyz^$third-party
 ||app.gelirartisi.com^$third-party
 ||vvhatsappsohbetim.site^
+||jubnaadserve.com^$third-party
```

Comment:
It was added to the Base filter https://github.com/AdguardTeam/AdguardFilters/commit/9ec8492dd965b7854ec1e21752ff527aa2989082, just remove it